tests/spec/core/data-cite-spec.js

```diff
@@ -263,6 +263,55 @@ describe("Core — data-cite attribute", () => {
     });
   });
 
+  it("Adds title to a reference when inline-link is empty normative reference", async () => {
+    const ops = {
+      config: makeBasicConfig(),
+      body: `${makeDefaultBody()}
+        <section>
+          <p id="t1"><a data-cite="!WHATWG-HTML"></a></p>
```

Comment:
Let's try just HTML. We also don't need the !.  

```suggestion
          <p id="t1"><a data-cite="HTML"></a></p>
```